build.js

```diff
@@ -35,6 +35,7 @@ redirects.push('/Unipool https://etherscan.io/address/0x48D7f315feDcaD332F68aafa
 redirects.push('/Curvepool https://etherscan.io/address/0x13B54E8271B3e45cE71D8f4fC73eA936873a34fC 302');
 redirects.push('/KybersUSD https://etherscan.io/address/0x4cb01bd05e4652cbb9f312ae604f4549d2bf2c99 302');
```

Comment:
Could you update Kyber to using the full checksum address? 0x4Cb01bd05E4652CbB9F312aE604f4549D2bf2C99

Reply:
and the oracle and the fee address (those are my fault sorry)